### DISCUSSION TITLE
How can I customize this template for myself and install locally?

### DISCUSSION THREAD

**Discussion by ageisen2000** [jasontaylordev/CleanArchitecture#517] [Q&A]
https://docs.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-item-template

According to this doc there is supposed to be a template file here somewhere and I'm missing it in this repo. Is this something that @jasontaylordev handles locally built off of this source code? 

I tried packing the WebUI project and installing and that doesn't work either... Not that I actually thought it would :D

**Comment by bramburn** [comment]
sorry what do you mean? i don't understand, are you unable to use the template using the dotnet cli?

**Reply by Poltergeisen** [reply]
In this specific instance I was told to manually apply a patch to the repo here until @jasontaylordev updated the public nuget. To make a local version of this to use with my local dotnet cli I was hoping we could include the necessary json files so that I could install my patch and test locally. Making it easier to test and do PRs

**Reply by bramburn** [reply]
ahh ok that sounds interesting i would like to do the same as I don't use IdentityServer, and would need to build your own nuget package, I am assuming you have to reconfigure it https://docs.microsoft.com/en-us/shows/nuget-101/create-and-publish-a-nuget-package-with-visual-studio-4-of-5
and probably host your own nuget somewhere.
